sklearn/svm/tests/test_svm.py

```diff
@@ -439,6 +439,76 @@ def test_sample_weights():
     assert_array_almost_equal(dual_coef_no_weight, clf.dual_coef_)
 
 
+@pytest.mark.parametrize(
+    "CLF, params",
```

Comment:
You are also testing some regressor, you should name this variable `Estimator`

Reply:
Right. Done

Reply:
Also added the three new tests:
test_svm_classifier_sided_sample_weight
test_svm_regressor_sided_sample_weight
test_svm_equivalence_sample_weight_C